docs/documentation/siddhi-4.0.md

```diff
@@ -1038,18 +1029,18 @@ select <event reference>([event index])?.<attribute name>, ...
 insert into <output stream>
 ```
 
-Keywords such as `and`, `or`, or `not` can used to illustrate the logical relationship.
+Keywords such as `and`, `or`, or `not` can be used to illustrate the logical relationship.
 
 Key Word|Description
 ---------|---------
 `and`|This allows both conditions of `and` to be matched by two events in any order.
-`or`|The state will succeed if either condition of `or` is satisfied. Here the event reference of the other condition will be `null`.
-`not <condition1> and <condition2>`| When `not` with `and` is present, it let you identify an event matching `<condition2>` before any event match the `<condition1>`. 
-`not <condition> for <time period>`| When `not` is present with `for`, it let you identify no event occurring that matches `<condition1>` for the given time  `<condition1>`.  is used to define a time period. E.g.`from not TemperatureStream[temp > 60] for 5 sec`. 
+`or`|The state succeeds if either condition of `or` is satisfied. Here the event reference of the other condition is `null`.
+`not <condition1> and <condition2>`| When `not` is included with `and`, events that match `<condition2>` that arrive before any event that matches `<condition1>` are identified. 
```

Comment:
its batter if we say... 
It identifies the event that match `<condition2>` arriving before any event that match `<condition1>` is identified. 
WDYT?

Reply:
Corrected.